src/reducers/index.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers, createStore } from 'redux';
+import booksReducer from './books';
+
+const store = createStore(
+  combineReducers({ books: booksReducer }),
```

Comment:
- [ ] [REQUIRED] It will be nice if this file combine  just reducer ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/ReducxReducers/CombinesAppReducers.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/)) as books' initial state will be passed from `src/index.js.`.

Reply:
Issue fixed